### PR TITLE
Request to merge Feature style

### PR DESCRIPTION
**Pull request by Jmagero**
-  Add CSS styles to the application to make it match this wireframe:

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,9 +1,45 @@
 body {
-  margin: 0 auto;
-  width: 50%;
+  /* margin: 0 auto;
+  width: 80%; */
```
Comment (umararshadbut) [inline style.css]: **[REQUIRED]** Please remove the commented code.

**Review by umararshadbut — CHANGES_REQUESTED** [review]
Hi @Jmagero and @kimothokamau , 

Good job so far :+1:!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- **[OPTIONAL]** Please create an assets directory and move the JavaScript and CSS files into the assets.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
HAPPY CODING :computer:
**Umair Arshad**

**Comment by kimothokamau** [comment]
- Removed commented code
- Put css and js files in assets folder

**Review by carlos-ssh — APPROVED** [review]
Hi @kimothokamau & @Jmagero, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**